modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -145,7 +145,22 @@ Configuration example:
 ----
 | No
 | `target` | The CloudHub 2.0 target name to deploy the app to. +
-Specify either a shared space or a private space available in your Deployment Target values in CloudHub 2.0. See xref:cloudhub-2::ch2-features.adoc[Features of CloudHub 2.0] for a detailed description on shared and private spaces. | Yes
+Specify either a shared space or a private space available in your Deployment Target values in CloudHub 2.0. Below are target values for shared space:
```

Comment:
Shared spaces, other than that in the default region, are available only for customers with `globalDeployment` entitlement enabled

Reply:
@juanmanuelmenendez - Not sure what do you mean y "for customers with globalDeployment entitlement enabled", can you elaborate more?